src/main/java/org/embulk/output/SnowflakeOutputPlugin.java

```diff
@@ -220,6 +231,35 @@ public ConfigDiff transaction(
     return configDiff;
   }
 
+  private void runDropStage(
```

Comment:
復帰処理も含んでのものだと思うので、名前に含んでおきたい気がしました

```suggestion
  private void runDropStageWithRecovery(
```

Reply:
こちら、trocco.io の外からPR出すとCIでエラーになるので、trocco.io内でブランチ切り直してPRだしました。
https://github.com/trocco-io/embulk-output-snowflake/pull/82

指摘の修正も含まれています 🙇 
https://github.com/trocco-io/embulk-output-snowflake/pull/82/commits/ed4f32052a56b140de83fb0f50db1622904d6c23